src/SSW.Rewards/Pages/LeaderboardPage.xaml

```diff
@@ -108,43 +109,14 @@
                 </ImageButton>
                 
             </Grid>
-
-            <Grid Grid.Row="1"
-                  ColumnDefinitions="*, 50"
-                  VerticalOptions="Center"
-                  HeightRequest="45"
-                  Padding="0,10">
-                <BoxView Grid.Column="0"
-                         Grid.ColumnSpan="2"
-                         BackgroundColor="{StaticResource TabBarBackground}"
-                         HorizontalOptions="FillAndExpand"
-                         VerticalOptions="CenterAndExpand"
-                         CornerRadius="10"/>
-
-                <controls:BorderlessEntry Grid.Column="0"
-                                          x:Name="SearchEntry"
-                                          Margin="10,0,0,0"
-                                          Text="{Binding SearchBarText}"
-                                          TextColor="White">
-                    <controls:BorderlessEntry.Behaviors>
-                        <behaviors:EventToCommandBehavior Command="{Binding SearchTextChanged}"
-                                                          CommandParameter="{Binding Text, Source={x:Reference SearchEntry}}"
-                                                          EventName="TextChanged" />
-                    </controls:BorderlessEntry.Behaviors>
-                </controls:BorderlessEntry>
-
-                <Label Grid.Column="1"
-                       FontFamily="{StaticResource FluentIcons}"
-                       Text="{Binding SearchBarIcon}"
-                       HorizontalOptions="Center"
-                       VerticalOptions="Center"
-                       FontSize="20"
-                       TextColor="White">
-                    <Label.GestureRecognizers>
-                        <TapGestureRecognizer NumberOfTapsRequired="1" Command="{Binding CancelSearchCommand}"/>
-                    </Label.GestureRecognizers>
-                </Label>
-            </Grid>
+            
+            <controls:Search
+                Grid.Row="1"
+                VerticalOptions="Center"
+                BorderColor="{StaticResource TabBarBackground}"
+                BackgroundColor="{StaticResource MainBackground}"
+                TextColor="White"
+                SearchTextChanged="Search_OnSearchTextChanged"/>
```

Comment:
Technically, `EventToCommandBehaviour` should work here, but it doesn't. The `Command` is `null` when the `SearchTextChanged` event fires.

Reply:
Fixed by creating a Bindable Property for Command.